libfreerdp/core/nla.c

```diff
@@ -2451,6 +2451,7 @@ void nla_free(rdpNla* nla)
 	sspi_SecBufferFree(&nla->ClientNonce);
 	sspi_SecBufferFree(&nla->PublicKey);
 	sspi_SecBufferFree(&nla->tsCredentials);
+	sspi_SecBufferFree(&nla->negoToken);
```

Comment:
ouch, this will break redirect/gateway related stuff

Reply:
can you explain?
this buffer shouldn't be freed when the whole rdpNla* struct is freed?

Reply:
BTW I checked and xfreerdp ables to be redirected after this change, i didn't see anything wrong

Reply:
@kubistika no, this is most of the time just a pointer to another buffer and this will be a double free

Reply:
okay, reverted